storage/Utils/Logger.cc

```diff
@@ -113,9 +132,9 @@ namespace storage
 
 
     Logger*
-    get_logfile_logger()
+    get_logfile_logger(const std::string& filename)
     {
-	static LogfileLogger logfile_logger;
+	static LogfileLogger logfile_logger(filename);
```

Comment:
AFAIS this will only respect the filename the first time the function is called. Keeping a list of LogfileLogger would be possible but would be over-engineering from my point of view. Documenting the behaviour should be enough.